pkg/apis/configuration/v1beta1/kongconsumergroup_types.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2021 Kong, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
```

Comment:
Perhaps something to discuss: should we introduce it in `v1alpha1` package first and graduate it to beta in the next minor (2.12)?

Reply:
Good question, for this CRD I expect in case of need only additional changes in terms of schema. We have some CRDs already graduated to `v1` that we'll make further changes without bumping. Do we have any established guidelines?

Reply:
> Do we have any established guidelines?

Not really. I don't think we've even actually bumped any of our CRD versions yet.

I don't think we expect changes (:crossed_fingers:) beyond the proposed 3.4 gateway implementation that'd require breaking changes for the CRD (maybe they do add a priority field, but we can stuff that in without affecting others), so I'd say just use v1beta1 to save us the work of bumping it without actually changing anything.